screens/score_board.py

```diff
@@ -15,8 +16,8 @@
 def score_board():
     run = True
 
-    score_title_font = pygame.font.Font(os.path.join(FONT_PATH, 'edit_undo.ttf'), 50)
-    score_font = pygame.font.Font(os.path.join(FONT_PATH, 'neue.ttf'), 35)
+    score_title_font = pygame.font.Font(resource_path(os.path.join(FONT_PATH, 'edit_undo.ttf')), 50)
+    score_font = pygame.font.Font(resource_path(os.path.join(FONT_PATH, 'edit_undo.ttf')), 35)
```

Comment:
I think here too the neue font looks better

Reply:
same as previous comment.